src/robot/parsing/lexer/settings.py

```diff
@@ -109,7 +110,8 @@ def _lex_error(self, setting, values, error):
     def _lex_setting(self, setting, values, name):
         self.settings[name] = values
         # TODO: Change token type from 'FORCE TAGS' to 'TEST TAGS' in RF 7.0.
-        setting.type = name.upper() if name != 'Test Tags' else 'FORCE TAGS'
+        setting_type_map = {'Test Tags': "FORCE TAGS", "Name": "SUITE NAME"}
+        setting.type = setting_type_map.get(name, name.upper())
```

Comment:
- Nice way to handle the fact that the setting name and type don't match.
- For consistency it would be good to use either single or double quotes in the dictionary. We've generally used single quotes (and ought to even mention that in contribution guidelines).

Reply:
switched to single quote